src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];

```

Comment:
The size of the Figure array is hardcoded. Consider using a constant or a variable to set the size of the array, making it easier to adjust later.